src/karthik/group-people-data/groupPeopleData.ts

```diff
@@ -0,0 +1,44 @@
+interface People {
+  _id: string;
+  name: string;
+  gender: string;
+  email: string;
+  phone: string;
+  city: string;
+}
+
+interface GroupOfPeopleByCity {
+  count: number;
+  city: string;
+  people: Array<People>;
+}
+
+const groupPeopleByCity = (
+  arrayOfObject: Array<any>
```

Comment:
I think we shall do type check with `Array<People>` instead of `Array<any>`. If not here, the input array of people structure shall be validated with it's format somewhere.

Reply:
fixed.